docs/CRSF.md

```diff
@@ -20,18 +24,20 @@ Note: An ArduPilot flight controller is assumed. For PX4 and iNav it needs to be
 - Tx Ser Link Mode = mavlink
 - Tx Snd RadioStat = off (yes, off!)
 
-Note: There are situations in which it can be usefull to enable "Tx Snd RadioStat", but you should do this only if you know what you are doing. You really should not need it for this setup.
+Notes: 
+- There are situations in which it can be useful to enable "Tx Snd RadioStat", but you should do this only if you know what you are doing. You really should not need it for this setup.
+- It is recommended to set Tx Ser Baudrate to 57600, as that will provide enough speed for all three operation modes (19 Hz, 31Hz, 50Hz).
```

Comment:
in contrast to the situation for Rx, for Tx the choice of baudrate is quite uncritical, and one also equally could choose 115200, or anything one likes. It's really just a matter fo what the user prefers.
So, I would change that note, maybe to
"The choice of Tx Ser Baudrate is uncritical and really determined by the user's need. It is recommended to set it to 57600 or higher, as this will provide enough speed for all operation modes (19 Hz, 31 Hz, 50 Hz)."

Reply:
Updated the note to this language.